src/citrine/resources/predictor_evaluation_workflow.py

```diff
@@ -53,3 +54,36 @@ def delete(self, uid: Union[UUID, str]) -> Response:
         """Predictor Evaluation Workflows cannot be deleted; they can be archived instead."""
         raise NotImplementedError(
             "Predictor Evaluation Workflows cannot be deleted; they can be archived instead.")
+
+    def create_default(self, predictor_id: UUID) -> PredictorEvaluationWorkflow:
+        """[ALPHA] Create a default predictor evaluation workflow for a predictor and execute it.
+
+        The current default predictor evaluation workflow performs 5-fold, 1-trial cross-validation
+        on all valid predictor responses. Valid responses are those that are **not** produced by the
+        following predictors:
+
+        * :class:`~citrine.informatics.predictors.generalized_mean_property_predictor.GeneralizedMeanPropertyPredictor`
+        * :class:`~citrine.informatics.predictors.ingredient_fractions_predictor.IngredientFractionsPredictor`
+        * :class:`~citrine.informatics.predictors.ingredients_to_simple_mixture_predictor.IngredientsToSimpleMixturePredictor`
+        * :class:`~citrine.informatics.predictors.label_fractions_predictor.LabelFractionsPredictor`
+        * :class:`~citrine.informatics.predictors.molecular_structure_featurizer.MolecularStructureFeaturizer`
+        * :class:`~citrine.informatics.predictors.simple_mixture_predictor.SimpleMixturePredictor`
+
+        If there are no valid responses, a default workflow is not created.
+
+        Parameters
+        ----------
+        predictor_id: UUID
+            Unique identifier of the predictor used to create a default workflow
+
+        Returns
+        -------
+        PredictorEvaluationWorkflow
+            Default workflow
+
+        """  # noqa: E501
+        url = self._get_path('default')
+        ref = PredictorRef(str(predictor_id))
```

Comment:
Why do we want to wrap this up in a PredictorRef rather than using UUID directly?

Reply:
I didn't know we could do that. If the backend expects a `PredictorRef` and I use the UUID directly with `self.session.post_resource(url, predictor_id)`, will the backend handle it correctly?

Reply:
That results in `TypeError: Object of type UUID is not JSON serializable`, but maybe if I convert to a string first?

Reply:
Nope, casting to a `str` returns a 400. I'm not sure what you mean by use it directly. If you're asking why the backed expects a `PredictorRef`, it was just for consistency with the `ModuleRef` that we use elsewhere.

Reply:
I don't think the backend distinguishes between uid types, but I see that this backend endpoint is expecting a post body of the form `{"predictor_id": "c7e50770-c22a-4065-9578-9761ebdfc77b"}`. Which you can get by doing `ref.dump()` if ref is a PredictorRef, or by being more explicit and saying something like `data = self.session.post_resource(url, {"predictor_id": predictor_id})` if `predictor_id` is the relevant id (or something similar, you might need to dump the `UUID` anyway?).

I would prefer not creating a new reference class for this, since I think the analogous ModuleRef has caused some confusion, and it doesn't seem to add anything other than this dict wrapping, but I might be missing something!

Reply:
oh, i was thinking something like `self.session.post_resource(url, {"predictor_id": predictor_id.dump()})` (commented before i saw the rest of your replies)

Reply:
That's fine. I don't have a strong preference either way (especially since this is the only place it's used). I deleted it in the latest commit, and now I create the payload manually like you suggested.